.gitignore

```diff
@@ -11,3 +11,5 @@ target/
 *.pdb
 .env
 todo.md
+
+.env
```

Comment:
The .env is already included in the .gitignore file (line 12). Please revert this change.

Reply:
Ah, my bad, will remove